src/Apps/Feature/Components/FeatureFeaturedLink/FeatureFeaturedLink.tsx

```diff
@@ -0,0 +1,91 @@
+import React from "react"
+import styled from "styled-components"
+import { space, SpaceProps } from "styled-system"
+import { Box, Sans, Serif, Image, Flex } from "@artsy/palette"
+
+const Container = styled.a`
+  ${space}
+  display: flex;
+  text-decoration: none;
+  flex-direction: column;
+`
+
+const Figure = styled(Box)`
+  position: relative;
+`
+
+const Cover = styled(Image)`
+  display: block;
+  width: 100%;
+  height: 100%;
+`
+
+const Title = styled(Sans)`
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  background: linear-gradient(
+    rgba(255, 255, 255, 0) 0%,
+    rgba(0, 0, 0, 0.25) 100%
+  );
+`
+
+const Meta = styled(Flex).attrs({ mt: 1 })``
+
+const MetaCell = styled(Flex)`
+  flex-basis: 50%;
+`
+
+export type FeatureFeaturedLinkProps = SpaceProps & {
+  wide?: boolean
+  featuredLink: {
+    title: string
+    href: string
+    subtitle?: string
+    description?: string
+    image: {
+      width: number
+      height: number
+      url: string
+    }
+  }
```

Comment:
Will just be the fragment.

Reply:
The codebase generally uses interfaces for objects / components -- we had a lint rule in place with TSLint that we need to update (mind updating in this PR?) 

The use of `type` is mostly constrained to one-liners.

Reply:
Ah interesting — I've been generally using it with styled-system props because it's much simpler to compose prop types. `type FooComponentProps = BoxProps & FlexBasisProps & { whatever: string }` — like in this instance I'm intersecting with SpaceProps, but would also probably want the StyledLinkProps or whatever since it's rest spread. This is also one of those things that doesn't matter much to me, whatever avoids bike shedding.